alacritty/src/input.rs

```diff
@@ -713,6 +713,8 @@ impl<T: EventListener, A: ActionContext<T>> Processor<T, A> {
             .mode()
             .contains(TermMode::ALT_SCREEN | TermMode::ALTERNATE_SCROLL)
             && !self.ctx.modifiers().state().shift_key()
+            && !self.ctx.terminal().mode().contains(TermMode::VI)
```

Comment:
I understand search, but why not during vi mode?

Reply:
Actually even during search I'm not sure it's necessary to disable this? Did you run into issues?

Reply:
Yeah, I tried to scroll in my irc client during VI mode, and nearly pasted some arbitrary string, because it was sending arrow key.

Reply:
That's more about mouse mode than anything else though, right?

Reply:
It was alternate scroll though, I'm still not sure why we keep it in Vi mode.

Reply:
Why wouldn't you? Only because vi mode is active, doesn't mean you don't want to scroll?

Reply:
Well, yeah, but Vi mode itself won't scroll and the scrolling is not native, it's just we try to send `Arrow` keys. At this point could just sent arrow keys directly.

Reply:
I don't see why we'd change anything here. Why wouldn't we send arrow keys, that's what we send without vi mode too.

Reply:
You can scroll the history while vi mode is active, so I don't see why alternate scroll wouldn't work.

Either both should work or neither.